android/pytorch_android/build.gradle

```diff
@@ -85,8 +85,7 @@ task sourcesJar(type: Jar) {
 }
 
 def getLibtorchHeadersDir() {
-  def abi = ABI_FILTERS.split(",")[0]
-  return "$rootDir/../build_android_$abi/install/include";
+  return property('LIBTORCH_HEADERS')
```

Comment:
Took a closer look, maybe if we change the default value to 
`pytorch_android/src/main/cpp/libtorch_include/$abi` it will work for all cases? as it's a reasonable assumption to make that symbolic link to build_android/install/include must have been set at earlier build steps...

Sorry I didn't see this earlier - it's an optional change :)